### ISSUE TITLE
Netfix 1080p extenstion on Windows Xp Chrome 49

### ISSUE THREAD

**Issue by ahm5jnhi5** [truedread/netflix-1080p#48]
I'm trying to get netflix to work on Windows Xp in Google Chrome in 1080p.
The last working version of Chrome on Windows Xp is Version 49.0.2623.112.
I have successfully managed to get netflix to play normally by copying the Widevine files to the WidevineCDM directory.
Now I'm trying to get it to work in 1080p.
The Netflix 1080p extension in this version of Chrome (49.0.2623.112) doesn't work.
When playing a video it just displays a black screen and the video doesn't play at all.
After removing the extension it plays fine in 720p.

So, is there a way to get the extension to work in this version of Chrome?
Same thing happens when trying the extension in this Chrome version on Windows 10.
So, the problem is from Chrome 49 not Windows Xp. 
I'm thinking, probably, a simple change to the extension is all that is needed.

**Comment by ItsNickBarry** [comment]
See the [release notes](https://github.com/truedread/netflix-1080p/releases).

**Comment by NuclearCow** [comment]
dayum windows XP xD Why not change to W10

**Comment by ahm5jnhi5** [comment]
heh.
Yeah.., I actually prefer working with XP.
After working with Win7 for so long then after it Win10, I finally had enough.
Win7 and especially Win10 tend to do things for you without you knowing, and I find that really annoying.
So, currently I'm trying as much as possible to make all my work on windows XP.

**Comment by hevanaa** [comment]
Impressive that someone is still using XP. When Vista came out, with all its problems, I switched to Linux and never looked back. Be aware that nobody checks for compatibility of their software on XP anymore.

**Comment by ahm5jnhi5** [comment]
To make things easier, here's a [link](http://cloud2.zoolz.com/s-G8tivbES) to download the Chrome version 49 standalone installer:
"Google_Chrome_(32bit)_v49.0.2623.112_(XP).exe"
This installer will install Chrome 49 without connecting to the internet or showing any dialogs.

I've also attached the Widevine files:
"[1.4.8.zip](https://github.com/truedread/netflix-1080p/files/2834620/1.4.8.zip)"
The folder "1.4.8.903" needs to be copied to this directory for netflix to work:
For Windows XP:
"C:\Documents and Settings\ _UserName_ \Local Settings\Application Data\Google\Chrome\User Data\WidevineCDM"
For Windows 10:
"C:\Users\ _UserName_ \AppData\Local\Google\Chrome\User Data\WidevineCDM"



I do use Ubuntu sometimes, for some things.
It's great how lightweight it is compared to Win10, but the UI is too simplistic for me to make it my main OS.
Plus that I have too much windows software that I use..
I'm more comfortable with the functionalities and controllability of Windows XP, so as long as something works in XP I wouldn't boot in another OS.
I am not really concerned about security, as I just try to be-careful online.
Even when using windows 10 I usually disable the antivirus (from registry).

I don't think editing the Netflix 1080p extension to work for Chrome 49 would be too complicated, so I really do hope that _truedread_ sees into this.

**Comment by ItsNickBarry** [comment]
Support for such an old operating system or browser version is unlikely.  Maybe the Firefox version will work for you: https://github.com/vladikoff/netflix-1080p-firefox

**Comment by ahm5jnhi5** [comment]
Same thing with the firefox addon. Doesn't work with firefox version 52.9 Esr.
Thanks to this [thread](https://msfn.org/board/topic/177768-netflix-playback/), I managed to get Netflix to work in XP in 720p using FireFox with Silverlight.
But the addon doesn't do anything and it still just plays in 720p.
I've posted an issue there also.

**Comment by ahm5jnhi5** [comment]
I guess I was a bit too naive to expect to get a solution for XP. But it was worth a shot.
I will see if I can figure out the fix when I find time.

About WannaCry, Microsoft actually made an exception and released a security update specifically for it on Windows XP.
Generally speaking I can see how it won't be a good idea to use XP online as time passes, mainly cause some sites won't be working as well in older browser versions..
But I will still be using XP offline.

I never knew Linux had builds that replicate Windows XP ^_^. That looks pretty interesting!
I'll definitely be checking them out..

Oh no Wine did not work for me. It's a better idea to use VirtualBox. That's what I've been doing for a while, since Windows XP can't be installed on new hardware.
But recently I finally finished creating a customised desktop build, by purchasing the motherboard and the other parts that are compatible with Windows XP.
And I'm finally able experience XP with it's full capabilities.

So, basically right now, when I want to use XP full I use my customised desktop.
And when I work on my laptop, I use Ubuntu MATE in conjunction with Virtual Windows XP.
I recently tumbled on Ubuntu MATE and turned out to be pretty amazing, customizable,  and comfortable to work with it with Virtual XP.

I will be closing this issue soon then.
